.pipelines/singletenancy/aks-swift/e2e-job-template.yaml

```diff
@@ -6,9 +6,10 @@ parameters:
 stages:
   - stage: ${{ parameters.name }}
     displayName: E2E - ${{ parameters.displayName }}
+    lockBehavior: sequential
```

Comment:
What does this do?

Reply:
with the swift e2e, we only have one cluster that we deploy cns and cni to. currently, the build pool acts as a semaphore to that cluster - the pool is configured to only have one machine, so only one pipeline can run the swift e2e at a time.
this change moves the "access control" into the pipeline. instead of implicitly locking by having to wait on the one machine, we explicitly lock and require the pipelines to execute this stage sequentially (in the order they are started). 
this _should_ let us expand the build pool for this stage so that we don't have to wait on a single machine to be provisioned and torn down every run (which is the longest part of this test). 

BUT that will require further work, at the moment this just explicitly locks this stage.